picard/_tools.py

```diff
@@ -89,3 +89,13 @@ def _ica_par(X, fun, max_iter, w_init, verbose):
     if verbose:
         print('Running Picard...')
     return W
+
+
+def fuse_mask(mask1, mask2):
```

Comment:
This is much much faster in numba. This is not really a bottleneck of the algorithm though.